Formula/c/cargo-fuzz.rb

```diff
@@ -0,0 +1,19 @@
+class CargoFuzz < Formula
+  desc "Command-line helpers for fuzzing"
+  homepage "https://rust-fuzz.github.io/book/cargo-fuzz.html"
+  url "https://github.com/rust-fuzz/cargo-fuzz/archive/refs/tags/0.12.0.tar.gz"
+  sha256 "d7c5a4589b8b5db3d49113e733553c286ed8b50800cbdb327b71a1c1f7c648f0"
+  license "Apache-2.0"
```

Comment:
https://github.com/rust-fuzz/cargo-fuzz#license:

> `cargo-fuzz` is distributed under the terms of both the MIT license and the Apache License (Version 2.0).

```suggestion
  license all_of: ["Apache-2.0", "MIT"]
```